plugin/test/fixtures/v5/with-adapters/e2e-tests/test-helpers.js

```diff
@@ -0,0 +1,277 @@
+/* eslint-disable no-unused-vars */
+const fetch = require(`node-fetch`)
+const { readFileSync } = require('fs')
+const path = require('path')
+
+const FormData = require('form-data')
+// Based on Gatsby Functions integration tests
+// Source: https://github.com/gatsbyjs/gatsby/blob/master/integration-tests/functions/test-helpers.js
+
+exports.runTests = function runTests(env, host) {
+  jest.setTimeout(10_000)
+
+  async function fetchTwice(url, options) {
```

Comment:
Is this to ensure we hit the origin server?

Reply:
I didn't author the test suite so I'm not sure why it was done like that - it was copied from already existing suite in https://github.com/netlify/netlify-plugin-gatsby/tree/main/plugin/test/fixtures/v5/functions-without-gatsby-plugin/e2e-tests

Those test suites don't actually deploy, they are running `ntl dev`, and I just changed it to use `netlify serve` + 
skipped some tests here https://github.com/netlify/netlify-plugin-gatsby/blob/884f10de2396acfe69afc7ae404806425054cce6/plugin/test/fixtures/v5/with-adapters/e2e-tests/test-helpers.js#L54-L59 due to how actual platform differs from `netlify serve` 

just to show they are working when deployed:
[`/api/some whitespace`](https://64aff944e6e57b04751b4a7a--adapter-test-e2e.netlify.app/api/some%20whitespace)
[`/api/with-äöü-umlaut`](https://64aff944e6e57b04751b4a7a--adapter-test-e2e.netlify.app/api/with-äöü-umlaut)
[`/api/some-àè-french`](https://64aff944e6e57b04751b4a7a--adapter-test-e2e.netlify.app/api/some-àè-french)
[`/api/some-אודות`](https://64aff944e6e57b04751b4a7a--adapter-test-e2e.netlify.app/api/some-אודות)